boba/cli.py

```diff
@@ -16,15 +17,23 @@
               default='./template.py', show_default=True)
 @click.option('--out', help='Output directory',
               default='.', show_default=True)
-@click.option('--lang', help='Language, can be python/R [default: inferred from file extension]',
+@click.option('--lang', help='Language [default: inferred from file extension]',
               default='')
-def compile(script, out, lang):
+@click.option('--supported_langs', '-sl', 'supp_langs', help='json file with supported languages',
+              default=None)
+def compile(script, out, lang, supp_langs):
     """Generate multiverse analysis from specifications."""
 
     check_path(script)
 
     click.echo('Creating multiverse from {}'.format(script))
-    ps = Parser(script, out, lang)
+    
+    supported_langs = None
+    if supp_langs:
+        with open(supp_langs, 'r') as l:
+            supported_langs = json.load(l)
```

Comment:
These lines should happen in the parser class ... Here we might simply check if the file exists (using `check_path`).

But if you modify the API to use `BOBA_CONFIG`, we won't modify this file ..